demisto_sdk/commands/common/errors.py

```diff
@@ -851,6 +853,18 @@ def invalid_image_name():
         return "The image's file name is invalid - " \
                "make sure the name looks like the following: <integration_name>_image.png"
 
+    @staticmethod
+    @error_code_decorator
+    def image_is_empty(image_path: str):
+        return f'Image corresponding to image path {image_path} should not be empty for partner support level. ' \
```

Comment:
```suggestion
        return f'The author image in path {image_path} should not be empty. ' \
```
correct me if I'm wrong, not only for partners

Reply:
Correct. Fixing